examples/folium_examples.ipynb

```diff
@@ -165,10 +165,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": null,
    "metadata": {
     "collapsed": false
    },
+<<<<<<< HEAD
```

Comment:
It seems we have a conflict remaining here. Don't worry I will fix this in another PR.